### DISCUSSION TITLE
[const char*]Support of Chinese charaters?

### DISCUSSION THREAD

**Discussion by huangweiwu** [wjakob/nanobind#289] [Q&A]
When I call the bound function with char* argument, it seems that Chinese characters invokes garbled characters.
I'm wondering if I miss some config or it has not been supported.

**Comment by wjakob** [comment]
Are the Chinese characters in UTF8 encoding? Otherwise it won't work.

**Reply by huangweiwu** [reply]
It's GBK encoding. Any possible to support it?

**Reply by wjakob** [reply]
Nope, sorry. You will have to convert to UTF8.

**Reply by huangweiwu** [reply]
I detect the encoding from python string as below and the result is utf-8.
But the garbled characters problem still exists.
```
s = "测试"
print(chardet.detect(s.encode())['encoding'])
```
